core/src/main/java/com/epam/esm/entity/GiftCertificateAsOrderItem.java

```diff
@@ -0,0 +1,18 @@
+package com.epam.esm.entity;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+import java.math.BigDecimal;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+@ToString(callSuper = true)
+public class GiftCertificateAsOrderItem extends BaseEntity {
+    private String name;
+    private String description;
+    private BigDecimal price;
+    private Integer duration;
+    private Integer amount;
```

Comment:
private Integer amount; - For what do you use this field?

Reply:
User can buy several the same certificates